api-client/src/routes/RouteInformationTypes.js

```diff
@@ -63,4 +63,5 @@ export type RouteInformationType =
   | NewsRouteInformationType
   | SimpleCityContentFeatureType
   | EventsPoisRouteInformationType
-  | null
+
+export type NullableRouteInformationType = RouteInformationType | null
```

Comment:
Should we name this RouteInformationType and the above NonNullRouteInformationType?